### ISSUE TITLE
cache gradle seems no effect?

### ISSUE THREAD

**Issue by AlbertShen0211** [mingchen/docker-android-build-box#113]
docker run --rm -v `pwd`:/project  -v "$HOME/.dockercache/gradle":"/root/.gradle"   mingc/android-build-box bash -c 'cd /project; ./gradlew build'

I run above command, everytime  download gradle, how fix it?

**Comment by AlbertShen0211** [comment]
I want to cache gradle, next time no download

**Comment by master-bob** [comment]
Hi AlbertShen0211,

I'm not sure what the problem is and won't be able to dig in locally for some time.

In the meantime, what troubleshooting have you already attempted?

Off the top of my head, with regards to troubleshooting, I'd probably confirm that the folders are being made correctly on the host and in the container. There might be some issues with quotes or whatnot so I'd confirm thay the syntax is correct / try the other variations.

Let us know what's up. 🤞

**Comment by master-bob** [comment]
I was able to test this functionality locally and saw that the cache was working. I used the following commands:

```sh
docker run -it --rm -v `pwd`:/project -v "$HOME/.dockercache/gradle/":"/root/.gradle" mingc/android-build-box:1.25.0 bash -c 'cd /project; ./gradlew build'
```
```sh
 docker run --rm -v `pwd`:/project -v "$HOME/.dockercache/gradle":"/root/.gradle" mingc/android-build-box:1.25.0 bash -c 'cd /project; ./gradlew tasks'
```

both versions where able to cache the gradle installs. Note that I tested with a project which required gradle 6.1.1 and gradle 7.6. Once the appropriate gradle version was downloaded, subsequent runs used the already downloaded gradle version. That is gradle was cached and the next time it did not download. But gradle did download twice. So in four runs, the first run of 6.1.1 downloaded gradle and then the next run of 6.1.1 did not,  the same for 7.6.
